tutorcairn/__about__.py

```diff
@@ -1 +1,9 @@
-__version__ = "16.0.3"
\ No newline at end of file
+__version__ = "17.0.0"
```

Comment:
Please rebase on top of nightly. From then on there should be just the `__version__` variable here.